docs/data/data-grid/pages.ts

```diff
@@ -0,0 +1,55 @@
+const pages = [
+  {
+    pathname: '/x/react-data-grid',
+    title: 'Components',
+    icon: 'ToggleOnIcon',
+    children: [
+      {
+        pathname: '/x/react-data-grid',
+        subheader: '/x/react-data-grid/overview',
+        title: 'Overview',
+      },
+      { pathname: '/x/react-data-grid/demo' },
```

Comment:
Not sure about the `react-` prefix here
Could you explain why you added it instead of just `/x/data-grid/*` ?

Reply:
I believe that this benefits the SEO proposed by @oliviertassinari in https://www.notion.so/mui-org/Documentation-per-product-decision-time-d9347d7d37b641129d2fea12d68555d2#67b6d390ec7c4977b5cb4d942ba2ee6b

Reply:
But why `react-` on components and `mui-` on the apis ?
Just to have benefits for the  SEO for both words ?

Reply:
For `mui-`, I think it relates to the package or the folder in the repo:
- `/material/api/mui-material/button`
- `/material/api/mui-lab/button` (here `mui-lab` lives in material)

But this is an interesting question for `x` packages 🤔, `mui-` does not make sense here. @mui-org/x @oliviertassinari any thought?

Reply:
I have polished https://www.notion.so/mui-org/Documentation-per-product-decision-time-d9347d7d37b641129d2fea12d68555d2#67b6d390ec7c4977b5cb4d942ba2ee6b to be more in sync with the current direction. It covers why `react-`.

Regarding why `mui-` we have never really talked about it so far. For me, the idea was to match the code imports `import Button from '@mui/material/Button`. To be honest, there, If it was only about me, I think that I would have done a 1:1 mapping as close as possible with code, so https://mui.com/api/mui-material/button/. But skipping `mui-` or adding the product prefix makes sense too.

Reply:
In my opinion, the product prefix is already explicit enough and as a user, I don't really care about the package or folder (in the URL). So I am good if we keep the URL the same and just add the product prefix.

- `/material/api/button`
- `/x/api/data-grid/data-grid`
- `/x/api/data-grid/grid-api`

Reply:
@m4theushw @flaviendelangle is this format looks good to you (similar to existing URLs, just prepend with the prefix `x`)?
- /x/api/data-grid/data-grid
- /x/api/data-grid/grid-api

Reply:
The URL seems clean to me that way yes :+1:

Reply:
Would it be simpler by removing the /data-grid/ prefix? So using the previous example:

- /x/api/data-grid/
- /x/api/data-grid-overview!/

It would match what we do in the other "spaces".